x-pack/plugins/apm/public/plugin.ts

```diff
@@ -359,7 +361,17 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
           id: 'storage-explorer',
           title: apmStorageExplorerTitle,
           path: '/storage-explorer',
+          searchable: featureFlags.storageExplorerAvailable,
         },
+        // ...(featureFlags.storageExplorerAvailable
+        //   ? [
+        //       {
+        //         id: 'storage-explorer',
+        //         title: apmStorageExplorerTitle,
+        //         path: '/storage-explorer',
+        //       },
+        //     ]
+        //   : []),
```

Comment:
I assume this should be removed?

```suggestion
        // ...(featureFlags.storageExplorerAvailable
        //   ? [
        //       {
        //         id: 'storage-explorer',
        //         title: apmStorageExplorerTitle,
        //         path: '/storage-explorer',
        //       },
        //     ]
        //   : []),
```

Reply:
ups yes